docs/content/en/api/auth.md

```diff
@@ -161,5 +161,5 @@ export default function({ $auth }) {
 
 ## tokens
 
-**Token** and **Refresh Token** are available on `$auth.token` and `$auth.refreshToken`.
+**Token** and **Refresh Token** are available on `$auth.strategy.token.get()` and `$auth.strategy.refreshToken.get()`.
```

Comment:
If I remember well, here we are showing how to access the `Token` and `RefreshToken` instaces, which have getters and setters. So I believe we should point to `$auth.strategy.token` and `$auth.strategy.refreshToken` without calling the `get()` method, since it returns the token itself, not the `Token` instance :)

Reply:
hello joao, thanks for review & I've updated it.

Reply:
if there any comment please let me know, because it's my first time co create PR :), i need to learn more to be a  contributor. thanks a lot.

Reply:
You did everything fine 😃 
I'll just rename the PR to fix semantic.

One thing I learned by contributing to open source is to create new branches in my fork for each feature/fix I made. So that I can open PRs for each modification separately :)